File: multi-sample/muliti-sample-api/src/main/java/com/wan/multi/api/member/MemberController.java
package com.wan.multi.api.member;

import com.wan.multi.core.member.Member;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;


@RestController
@RequestMapping("/members")
public class MemberController {

    @GetMapping
    public Member test(){
        return new Member("test");
    }
}

File: kotlin-elk-example/api/src/main/kotlin/com/wan/elk/api/MemberController.kt
package com.wan.elk.api

import com.wan.elk.core.Member
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/members")
class MemberController {

    @GetMapping
    fun getMember(): Member {
        return Member("test", 123)
    }
}
